.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+_Add ticket reference to Pull Request title: e.g. 'FS-123: Add content', if there is no ticket prefix with BAU_
+
+
+### Change description
+_A brief description of the pull request_ 
+
+- [ ] Unit tests and other appropriate tests added or updated
+- [ ] README and other documentation has been updated / added (if needed)
+- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items to prevent them from overlapping the logo")
```

Comment:
Sorry to be pedantic, but could we maybe slightly change this example?

It's 74 characters long, but the maximum amount of characters in a title is 72 - so it might be truncated during some git commands.

Reply:
```suggestion
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")
```